web/src/settings_preferences.js

```diff
@@ -76,7 +76,12 @@ function org_notification_default_language_modal_post_render() {
             );
             $language_element.text(new_language);
             $language_element.attr("data-language-code", setting_value);
-            settings_components.save_discard_widget_status_handler($("#org-notifications"));
+            settings_components.save_discard_widget_status_handler(
+                $("#org-notifications"),
+                false,
+                undefined,
+                undefined,
+            );
```

Comment:
I those those last two parameters are meant to be optional -- with the default values used if not passed. I think possibly the type annotation for `for_realm_default_settings` should just have a default of false? Only one of the callers passes those extra parameters.

Reply:
Agreed. Addressed in [#29835(commit)](https://github.com/zulip/zulip/pull/29835/commits/5631aa7b087a7959b4264a8c562879f945b0e00a).